backend/shared/shared/audit_log/jobs/quater_hourly/send_audit_log.py

```diff
@@ -0,0 +1,12 @@
+from django.conf import settings
+from django_extensions.management.jobs import QuarterHourlyJob
+
+from shared.audit_log.tasks import send_audit_log_to_elastic_search
+
+
+class Job(QuarterHourlyJob):
+    help = "Send AuditLogEntry to centralized log center every 15 minutes"
+
+    def execute(self):
+        if settings.ENABLE_SEND_AUDIT_LOG:
+            send_audit_log_to_elastic_search()
```

Comment:
What happens if send_audit_log_to_elastic_search throws an exception? Maybe add the call in a try-catch block and log exception content?

Reply:
AFAIK, if the send audit log task fails, we want it to raise an error and that error will be captured to Sentry. I guess we just leave it to fail naturally here. There is nothing else we have to do rather than try to send the log in the next quarter-hour. Please correct me if I'm wrong :)